tests/console/yast2_samba.pm

```diff
@@ -230,9 +226,45 @@ sub run {
     send_key 'alt-o';
 
     wait_serial('yast2-samba-server-status-0', 60) || die "'yast2 samba-server' didn't finish";
+}
+
+sub gui_new {
+    assert_screen 'yast2_samba-workgroup';
+    send_key 'alt-n';
+    if (check_screen 'yast2_still_susefirewall2') {
+        record_soft_failure 'bsc#1064405';
+        send_key 'alt-c';
+    }
+    wait_still_screen;
+    # from here we have new workflow because of changes in sles 15
+}
+
+
+sub run {
+    setup_ldap;
+
+    # install samba stuffs at first
+    zypper_call("in samba yast2-samba-server");
+
+    # start samba server configuration
+    script_run("yast2 samba-server; echo yast2-samba-server-status-\$? > /dev/$serialdev", 0);
+    # check Samba-Server Configuration got started
+    if ((is_tumbleweed) || (is_sle && sle_version_at_least('15'))) {
```

Comment:
We use this ugly condition twice here, let's move it to the lib, and again we need to make it work on sle 15.

Reply:
we do care new TW and sles 15, leap is not important atm. the whole yast2 test group is not activated for leap.

Reply:
That's not true, we have leap tests running on o3. They are not in best shape, but it's mainly because we had no time to fix them. So I believe we have to at least define expected behavior, so only new set of needles is required in best case.

Reply:
atm o3 is not testing leap for yast2 at all. do we have check var like "is_leap"? 
if yes, I can add this however..